frontend/src/metabase/admin/upsells/components/UpsellCard.tsx

```diff
@@ -18,6 +18,20 @@ type UpsellCardProps = {
   children: React.ReactNode;
 };
 
+interface FullWidthVariant {
+  maxWidth?: never;
+  fullWidth?: boolean;
+}
+
+interface FixWidthVariant {
```

Comment:
tiny naming nit

```suggestion
interface FixedWidthVariant {
```

Reply:
Fixed in https://github.com/metabase/metabase/pull/45424/commits/06f55e011687acfa850fcb76a0811e7f3183c349